Src/Support/Google.Apis.Auth/OAuth2/ServiceCredential.cs

```diff
@@ -100,6 +109,10 @@ public Initializer(string tokenServerUrl)
         /// <summary>Gets the HTTP client used to make authentication requests to the server.</summary>
         public ConfigurableHttpClient HttpClient { get; }
 
+        internal IHttpClientFactory HttpClientFactory { get; }
```

Comment:
It's not clear to me why we need these. We're already copying the HttpClient; when would we use the factory again?

Reply:
Fairly sure `HttpClient` isn't being copied. The `ServiceCredential` ctor creates a new `HttpClient` from the initializers `HttpClientFactory`.
With the current behaviour the new (scoped) credential will always use the default `HttpClientFactory` to create the `HttpClient` regardless of whether a custom factory was specified in the origin credential.
This fixes that, but does now mean a custom factory will be called where it used not to be.
The fact that no-one has ever complained about this probably means that no-one is using custom factories.